test-vectors/vectors/tx_out_records/incorrect_tx_out_records.jsonl

```diff
@@ -1,10 +0,0 @@
-{"spurious_view_private_key_hex_proto_bytes":"0a20824c753f97cd96c94d707cdbb495b1cce2a8dfd0fca2a45a93543292245dae0c","tx_out_record_hex_proto_bytes":"11e8993af924d072341a204220d6bf425422ceca582788baa46379c12e19a5ceb6ea96fe2cdacbe537a7612220febd16fcd5c235b87565149b9a5e71dd05b2bedd19082567556d25b4101c7603296400000000000000310100000000000000390a00000000000000453841cad04a42d96fd3ae64b1ab61ef87706ecf47199183864205987b455e3224295e7e704343e8c3d96d8b6c0fa3dcb98d8cf51343141405bae3ffcfe881b1f6cc1b33572873e2475208cf29351517665a70"}
```

Comment:
guessing the removal of this file is causing the CI failures
I don't recall how/why but sometimes these get accidentally cleaned and not re-built